### PR TITLE
Feature/metrics

### PR DESCRIPTION
**Pull request by kiwiidb**
Add Prometheus middleware if configured

### CODE REVIEW

**Review by bumi — COMMENTED** [review]
We should also add some docs for this in the readme

File: lib/service/config.go [hunk 1]
```diff
@@ -15,4 +15,6 @@ type Config struct {
 	DefaultRateLimit      int    `envconfig:"DEFAULT_RATE_LIMIT" default:"10"`
 	StrictRateLimit       int    `envconfig:"STRICT_RATE_LIMIT" default:"10"`
 	BurstRateLimit        int    `envconfig:"BURST_RATE_LIMIT" default:"1"`
+	EnablePrometheus      bool   `envconfig:"ENABLE_PROMETHEUS" default:"false"`
+	PrometheusPort        int    `envconfig:"PROMETHEUS_PORT" default:"9092"`
```
[the remark this answers is not in the document]
Comment (kiwiidb) [reply lib/service/config.go:19]: I prefer this to be honest. Then the user doesn't have to look up what you are supposed to take.

[the remark this answers is not in the document]
Comment (bumi) [reply lib/service/config.go:19]: ok 👍

**Review by bumi — APPROVED** [review]
I'd still add a note to the README on what it is and how it can be used? (maybe just a link to an external doc or something)

**Comment by bumi** [comment]
👍

**Pull request merged**